src/drive/web/modules/views/Recent/index.jsx

```diff
@@ -26,7 +42,26 @@ export const RecentView = ({ router, children }) => {
     router.push(`/recent/file/${file.id}`)
   })
 
-  const actions = useActions({ hasWriteAccess: false, canMove: true })
+  const client = useClient()
+  const { pushModal, popModal } = useContext(ModalContext)
+  const { refresh } = useContext(SharingContext)
+  const dispatch = useDispatch()
+
+  const actionsContext = {
+    client,
+    pushModal,
+    popModal,
+    refresh,
+    dispatch,
+    router,
+    location,
+    hasWriteAccess: false,
+    canMove: true
+  }
```

Comment:
Nit: Since those are directly as props, I think this can be renamd into actionsProps. Similarly, at the implementation it, s/actionsOptions/actionsProps.